app/javascript/apps/Project/index.tsx

```diff
@@ -15,8 +15,9 @@ import { Project, Stage, Tags as TagsType } from "Typings"
 import { deployBlockPath, projectEditPath } from "../../shared/UrlHelper"
 import { formattedDependencies, formattedTags } from "../../shared/Helpers"
 import { MainLayout } from "../../components/MainLayout"
-import { StageWithComparison } from "../../components/Stage/StageWithComparison"
+import { ProjectMaintenanceRecomendations } from "../../components/Project/MaintenanceRecommendations"
```

Comment:
Typo (should be`ProjectMaintenanceRecommendations`) here and in a few related locations.